smoke-test.sh

```diff
@@ -161,6 +161,12 @@ function doCommand() {
             else
                 "${dotnetCmd}" $newArgs >> "$logFile" 2>&1
             fi
+            # XXX crummel XXX
```

Comment:
(Nit) IMO the username comments aren't needed with the issue link + git blame.

Reply:
Removed.